python/setup.py

```diff
@@ -0,0 +1,5 @@
+from setuptools import setup, find_packages
+
+setup(name='ingestion',
+      packages=find_packages(),
```

Comment:
Neat trick! There's some stuff in the README about putting a setup.py file in each package, can you update that?